resources/lang/en/journal.php

```diff
@@ -13,6 +13,7 @@
     'journal_add' => 'Add a journal entry',
     'journal_edit' => 'Edit a journal entry',
     'journal_empty' => 'Empty journal',
+    'journal_created_at' => 'Created at',
```

Comment:
You have to add a parameter to the text so it works on all languages.
```suggestion
    'journal_created_at' => 'Created at {date}',
```